css/index.css

```diff
@@ -0,0 +1,376 @@
+  margin: 0 auto;
+}
+.nav-bar {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #F6F6F6;
+  border-bottom: #F1F1F1;
+}
+.nav-bar h1 {
+  font-size: 4rem;
+  color: #23293B;
+}
+.nav-bar nav {
+  display: flex;
+  justify-content: center;
+  margin: 15px 2rem;
+}
+.nav-bar nav a {
+  color: black;
+  font-family: #454A59;
+  text-decoration: none;
+  font-size: 2rem;
+  margin: 10px 2rem;
+}
+@media (max-width:500px) {
+  .nav-bar nav a {
+    font-size: 1.5rem;
+  }
+}
+footer {
+  color: #22283B;
+  background: #f4f4f4;
+  padding: 20px 0;
+  border: 1px double #e0e0e0;
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.intro h2 {
+  padding: 15px;
+}
+.intro p {
+  padding: 0 15px;
+}
+.intro .header-image {
+  border-radius: 4px;
+}
+.space-images {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+@media (max-width:500px) {
+  .space-images .blast {
+    width: 92%;
+    margin: 5% 5%;
+  }
+}
+@media (max-width:500px) {
+  .space-images .sky,
+  .space-images .launch {
+    display: none;
+  }
+}
+.space-images img {
+  width: 28%;
+  margin: 20px 2.5%;
+  border-radius: 20px;
+  height: 100%;
+}
+@media (max-width:500px) {
+  .space-images img {
+    width: 92%;
+    margin: 0 5%;
+  }
+}
+.visit-planets {
+  padding: 15px;
+}
+.visit-planets h3 {
+  margin-top: 11px;
+}
+.visit-planets p {
+  margin-top: 13px;
+}
+.walk {
+  margin-top: 60px;
+  border-top: 1px solid #dee2e6;
+  padding: 50px 60px;
+  display: flex;
+  justify-content: center;
+}
+@media (max-width:500px) {
+  .walk {
+    flex-direction: column-reverse;
+    margin-top: 10px;
+    padding: 10px 5%;
+  }
+}
+.walk .walk-text {
+  width: 100%;
+}
+@media (max-width:500px) {
+  .walk .walk-text {
+    width: 100%;
+    margin-top: 10px;
+  }
+}
+.walk .walk-text h3 {
+  margin-bottom: 10px;
+}
+.walk .walk-text p {
+  margin-bottom: 26px;
+}
+.walk .walk-text .btn {
+  background-color: #22283A;
+  color: #F6F7F8;
+  width: 30%;
+  text-align: center;
+  border-radius: 5px;
+  padding: 5px 10px;
+  font-size: 1.5rem;
+}
+.walk .walk-text .btn:hover {
+  background-color: #F6F7F8;
+  color: #22283A;
+}
+@media (max-width:500px) {
+  .walk .walk-text .btn {
+    width: 100%;
+    padding: 20px 0;
+  }
+}
+.walk .walk-img {
+  width: 100%;
+  margin-left: 4%;
+}
+@media (max-width:500px) {
+  .walk .walk-img {
+    width: 100%;
+    margin-left: 0;
+    justify-content: center;
+  }
+}
+.walk .walk-img img {
+  width: 100%;
+  height: auto;
+  border-radius: 10px;
+}
+.blog {
+  margin-top: 60px;
+  border-top: 1px solid #dee2e6;
+  padding: 50px 60px;
+  display: flex;
+  justify-content: center;
+}
+@media (max-width:500px) {
+  .blog {
+    flex-direction: column-reverse;
+    margin-top: 10px;
+    padding: 10px 5%;
+  }
+}
+.blog .blog-text {
+  width: 100%;
+}
+@media (max-width:500px) {
+  .blog .blog-text {
+    width: 100%;
```

Comment:
width isn't needed here, since you already set it on the element outside the media query